chemprop/uncertainty/uncertainty_calibrator.py

```diff
@@ -377,21 +405,23 @@ def calibrate(self):
         individual_vars = np.array(
             self.calibration_predictor.get_individual_vars()
         )  # shape(models, data, tasks)
-        targets = np.array(self.calibration_data.targets())
+        targets = np.array(self.calibration_data.targets(), dtype=float)
+        mask = np.array(self.calibration_data.mask(), dtype=bool)
         errors = uncal_preds - targets
+        errors = np.where(mask, errors, 0.)
 
         def objective(scaler_values: np.ndarray):
-            scaler_values = np.reshape(softmax(scaler_values), [-1, 1, 1])
+            scaler_values = np.reshape(softmax(scaler_values), [-1, 1, 1])  # (models, 1, 1)
             scaled_vars = np.sum(
                 individual_vars * scaler_values, axis=0, keepdims=False
-            )
+            )  # (data, tasks)
             nll = np.log(2 * np.pi * scaled_vars) / 2 + (errors) ** 2 / (
                 2 * scaled_vars
             )
-            nll = np.sum(nll, axis=0)
+            nll = np.sum(nll)
```

Comment:
```suggestion
            nll = np.where(mask, nll, 0.)
            nll = np.sum(nll)
```
In addition to z and t scaling, this seems to be a similar problem.

Reply:
You are completely right @shihchengli

Reply:
I think this will necessitate changes to testing too. Will follow up.